azurelinuxagent/common/protocol/wire.py

```diff
@@ -1393,15 +1399,9 @@ def parse(self, xml_text):
         self.remote_access_uri = findtext(container, "RemoteAccessInfo")
         lbprobe_ports = find(xml_doc, "LBProbePorts")
         self.load_balancer_probe_port = findtext(lbprobe_ports, "Port")
-        return self
 
 
 class HostingEnv(object):
-    """
```

Comment:
many commens in this code were totally out of whack --- maybe copy and paste errors; removed them